xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/macros.vm

```diff
@@ -1563,6 +1563,8 @@ $Msz MB##
       #set ($filterType = 'text')
     #end
   #end
+  #set($title = $services.localization.render('platform.livetable.filtersTitle',
+    [$services.localization.render("${transprefix}$column")]))
```

Comment:
Since you want to remove the duplication, you could include the escaping here also, and have an ``escapedTitle`` variable.